examples/linking-contracts/contracts/TokenExchange.sol

```diff
@@ -1,37 +1,45 @@
-pragma solidity ^0.5.0;
+// contracts/TokenExchange.sol
+pragma solidity ^0.6.2;
 
+// Import base Initializable contract
 import "@openzeppelin/upgrades/contracts/Initializable.sol";
```

Comment:
Contracts eth-pkg contains its own version of `Initializable`, so to avoid the clash I think it would be best here to import that one. In the snippet here it doesn't clash because neither of the imported files use Initializable.
```suggestion
import "@openzeppelin/contracts-ethereum-package/Initializable.sol";
```